awx/main/tasks/system.py

```diff
@@ -475,8 +476,8 @@ def inspect_execution_nodes(instance_list):
                     execution_node_health_check.apply_async([hostname])
 
 
-@task(queue=get_local_queuename)
-def cluster_node_heartbeat():
+@task(queue=get_local_queuename, bind_kwargs=['dispatch_time', 'active_task_ids'])
+def cluster_node_heartbeat(dispatch_time=None, active_task_ids=None):
```

Comment:
I use the "bind" wording because Celery has a `bind` kwarg

https://docs.celeryq.dev/en/stable/userguide/calling.html

In celery, this attaches a worker instance. In our case, the parameters we really want are from a separate process, so we can't just pass python objects. They need to go through serialization and through the socket communication.

I also imagine that "publish_time" could be added to this easily. I don't have a solid use case for that, other than some logging.